### PR TITLE
[PT-D][Tensor Parallel] Add more test cases when we use use_orig_params for FSDP wrapping

### PR DESCRIPTION
**Pull request by fduwjj**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #89779

Differential Revision: [D41600656](https://our.internmc.facebook.com/intern/diff/D41600656)

### CODE REVIEW

File: torch/distributed/_tensor/ops/math_ops.py [hunk 1]
```diff
@@ -139,3 +140,35 @@ def var_correction_rule(op_schema: OpSchema) -> OutputSharding:
     return reduction_rule(
         op_schema, dims=dims, keep_dim=keep_dim, reduction_linear=False
     )
+
+
+@register_prop_rule("aten.native_layer_norm.default")
+def _prop_native_layer_norm(op_schema: OpSchema) -> OutputSharding:
```
Comment (wanchaol) [inline torch/distributed/_tensor/ops/math_ops.py:146]: can we delete this? This is actually right now an experimental op in https://github.com/pytorch/tau/blob/main/spmd/tensor/experimental_ops.py#L10

**Review by wanchaol — APPROVED** [review]
looks good to me, one comment about removing experimental op

File: torch/distributed/_tensor/ops/math_ops.py (hunk 1 above)
Comment (fduwjj) [reply torch/distributed/_tensor/ops/math_ops.py:146]: Sure, totally.

**Comment by fduwjj** [comment]
@fduwjj has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D41600656).

**Comment by fduwjj** [comment]
@fduwjj has imported this pull request.  If you are a Meta employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D41600656).

**Pull request closed without merging**